com.github.babluboy.nutty.json

```diff
@@ -0,0 +1,106 @@
+{
+    "id": "com.github.babluboy.nutty",
+    "runtime": "org.gnome.Platform",
+    "runtime-version" : "3.28",
+    "sdk": "org.gnome.Sdk",
+    "command": "com.github.babluboy.nutty",
+    "finish-args": [
+            "--share=ipc",
+            "--socket=x11",
+            "--device=dri",
+            "--socket=wayland",
+            "--filesystem=home",
```

Comment:
As it's a network utility application, how does it not require the network access?

Reply:
`--share=network`